### ISSUE TITLE
Can't install the package on Windows (python 3.11)

### ISSUE THREAD

**Issue by kindaway** [casdoor/casdoor-python-sdk#72]
```
Using version ^1.17.1 for casdoor                                                                                                    

Updating dependencies
Resolving dependencies...

Package operations: 1 install, 0 updates, 0 removals

  • Installing casdoor (1.17.1)

  at ~\AppData\Local\Programs\Python\Python311\Lib\site-packages\installer\utils.py:235 in parse_entrypoints
      231│
      232│         for name, value in config.items(section):
      233│             assert isinstance(name, str)
      234│             match = _ENTRYPOINT_REGEX.match(value)
    → 235│             assert match
      236│
      237│             module = match.group("module")
      238│             assert isinstance(module, str)
      239│
```

This error appears when I try to install `casdoor` on my Windows PC on python 3.11 via poetry.
If I try to install it using pip, I get this error instead:
```
ERROR: For req: casdoor. Invalid script entry point: <ExportEntry test = tests:None []> - A callable suffix is required. Cf https://packaging.python.org/specifications/entry-points/#use-for-scripts for more information.
```

Some issue with tests, perhaps? 🤔

**Comment by ganisback** [comment]
I had the same problem

**Comment by hsluoyz** [comment]
@kindaway @ganisback fixed in v1.17.2: https://github.com/casdoor/casdoor-python-sdk/releases/tag/v1.17.2
